base_location_geonames_import/wizard/geonames_import.py

```diff
@@ -25,13 +25,10 @@
 class CityZipGeonamesImport(models.TransientModel):
     _name = "city.zip.geonames.import"
     _description = "Import City Zips from Geonames"
-    _rec_name = "country_id"
```

Comment:
If the model doesn't have `name` field, it requires a  `_rec_name` of an existing field. If you don't want `country_id`, then put `country_ids`

Reply:
@pedrobaeza Thank you for your feedback. The change has been applied.